sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_connection.py

```diff
@@ -161,7 +163,10 @@ def __exit__(self, *args):
 
     def _set_state(self, new_state):
         # type: (ConnectionState) -> None
-        """Update the connection state."""
+        """Update the connection state.
+
+        :param ~._pyamqp.constants.ConnectionState new_state: The new state to transition to.
```

Comment:
I was using this as a placeholder for when we make it an external library so that it would be uniform across SB and EH and reduce merge conflicts when keeping the code consistent